generate.css.mjs

```diff
@@ -0,0 +1,36 @@
+import brand from "@neos-project/brand";
+
+const brandVars = Object.assign(
+    {
+        //
+        // Spacings
+        //
+        "--goldenUnit": "40px",
+        "--spacing": "16px",
+        "--halfSpacing": "8px",
+
+        //
+        // Sizes
+        //
+        "--sidebarWidth": "320px",
+
+        //
+        // Font sizes
+        //
+        "--baseFontSize": "14px",
+    },
+    brand.generateCssVarsObject(brand.config, "brand")
```

Comment:
im kinda scared that we seemingly have this `@neos-project/brand` thing exposed and i dont know who maintains that ^^ ... neos at least doesnt use it itself.